### DISCUSSION TITLE
Allow override of excluded search files (i.e force include them)

### DISCUSSION THREAD

**Discussion by eyalroth** [community/community#27474] [Other Feature Feedback, Questions, & Ideas] [answered]
Please add an option to override the exclusion of files to index, either on a repo or organization level.

There are already a few discussions about unexpected excluded files (#10361 #13919), and I just found another case of `config/test.json` file not being indexed/searched too.

In the aforementioned discussions @colinwm has referred users to [the FAQ](https://cs.github.com/about/faq#indexed-content), but it does not contain the list of excluded files.

Having these files excluded from search, while also not knowing which are these files excactly, creates a **dangerous** case of false negative searches when trying to investigate cross-repository issues.

**Comment by colinwm** [comment, marked as answer]
Totally agree that the surprising absence of files from the index is not acceptable, and we've gotten a lot of feedback on that. We are working hard on scaling our index to include all files, so that this won't be an issue. 

Unfortunately you'll have to wait a bit for updates on that.
